app/components/CompanyValues.tsx

```diff
@@ -0,0 +1,150 @@
+// LIBRARY IMPORTS
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  Icon,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import Image from "next/image";
+import { FaRegClock, FaPeopleGroup, FaRegStar } from "react-icons/fa6";
+
+// LOCAL IMPORTS
+import gearBackground from "../../public/images/machine-gear-background.png";
+
+// VALUES COMPONENT DEFINITION
+export default function CompanyValues() {
+  return (
+    <>
+      <Image
+        src={gearBackground}
+        placeholder="blur"
+        fill
+        sizes="100vw"
+        style={{
+          objectFit: "cover",
+          zIndex: -1,
+        }}
+      />
+      <Box
+        bgImage={gearBackground}
+        backgroundRepeat="no-repeat"
+        backgroundSize="cover"
+        p={10}
+        bg="brand.backgroundDark"
+        opacity="90%"
```

Comment:
Actually, I just discovered that you can even simply use 
```javascript
bg="rgba(11, 17, 62, 0.9)" p={10}
``` 
without providing image, because this box essentially creates an overlay over the image.

Reply:
Such an easy fix, thanks for putting in so much time to find it. Works great!